packages/dashboard-core-plugins/src/linker/LinkerLink.scss

```diff
@@ -12,89 +12,144 @@ $dash-delete-color-1: $danger;
 $dash-delete-color-2: mix($danger, $content-bg, 25%);
 $dash-size: 10;
 
-.linker-overlay svg {
-  .link-dot,
-  .link-triangle {
-    fill: $dash-color-1;
+.linker-overlay {
+  .btn-fab {
+    pointer-events: all;
```

Comment:
You'll want a `z-index: 20` here as well so buttons appear above any other links